hugo.toml

```
@@ -0,0 +1,3 @@
+baseURL = 'https://csmeyns.github.io/twcnl-test/'
```

```suggestion
baseURL = 'https://techwerkers.nl'
```

I suspect we will have to provide an override for this in the netlify preview site stuff. Will check that out after we merge.